galite-tests/src/test/kotlin/org/kopi/galite/tests/ui/vaadin/field/FieldsTests.kt

```diff
@@ -195,6 +209,24 @@ class FieldsTests : GaliteVUITestBase() {
     }
   }
 
+  @Ignore
```

Comment:
If the test doesn't execute, we can omit it altogether with the forms.

Reply:
we will fix it in the future